doc/content/getting-started/programming-software/st-examples/mounting/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "USB Mounting"
+description: ""
+weight:
+distributions: null
+---
+
+A ST-LINK can work similarly to a USB drive, where you can simply place data on from your computer. When you place a binary executable, it will program it onto the {{% gn %}} automatically. This technique is good for people that do not want to install any extra software.  
```

Comment:
Does this tutorial target GNSE programming? or all generic nodes?
If you are targeting GNSE, please adjust all {{% gn %}} in this PR to {{% gnse %}} unless you are trying to to target all future GN boards

Reply:
All nodes use the STM32WL chip, so I guess it targets all Generic Nodes.

Reply:
That is not true, GNTE uses another MCU